test/index.test.js

```diff
@@ -115,11 +115,31 @@ describe('Simplereach', function() {
       beforeEach(function() {
         analytics.stub(window.SPR, 'collect');
       });
+      var title = document.title;
```

Comment:
As a general convention we like to keep these kind of things scoped inside each unit test unless it is a general data fixture that we will use across multiple unit tests.

So for these purposes, I would just write the `.track()` call inside each unit test as you have already done but actually hardcode the property values rather than referencing outside variables :)